### ISSUE TITLE
Knowing the details of the model trained

### ISSUE THREAD

**Issue by Ankit-Vohra** [googlecreativelab/teachablemachine-community#189]
I'm using teachable to train a binary classifier, it is giving me some very good results so I exported the model and have deployed it.
When I load the model and print model.summary() I'm only able to go through 2 sequential layers.
![image](https://user-images.githubusercontent.com/44256054/113556129-a619f580-9619-11eb-9d91-c7dfb4cd291c.png)

Is there any way to identify the entire architecture of the model which is built on teachable that will be really helpful to those who want to dive deep into the model.
Thanks

**Comment by HalfdanJ** [comment]
Which type of model are you training? (image/audio/pose)

**Comment by Ankit-Vohra** [comment]
I'm training a Binary image classification
It's giving me very good accuracy so I want to know the entire architecture
of the model
I can share the teachable model file if required

**Comment by HalfdanJ** [comment]
The model is standard Mobilenet v2 without tops, and then an added a new top layer for the custom transfer learning. 
https://github.com/googlecreativelab/teachablemachine-community/blob/master/libraries/image/src/custom-mobilenet.ts#L303

**Comment by yo80106** [comment]
Hi, I am also interested in knowing more about the base model in Teachable Machine. I am wondering that how did you shrink down the size of mobilenet v2? I used "tf.keras.applications.mobilenet_v2.MobileNetV2" to import the MobileNet 2; however the total parameters is over 2 millions. Also, I think MobileNet is expected to accept the image in 3 channels (RGB), how did you make it to adapt grayscale images? Thank you in advance.

**Comment by HalfdanJ** [comment]
The model we load is an unmodified version of mobilenet v2 with alpha 0.35, image size 224. The model takes rgb. Then after truncaing the top layer, all we do is adding globalAveragePooling2d to the model in the code linked above. 

To convert to keras, we are using the tensorflowjs commandline utility, I dont know what that does in its internals, but I think it should just do a 1:1 conversion.

**Comment by ansariahmad** [comment]
@Ankit-Vohra 
Use this code
model.summary(expand_nested=True)
